lib/matplotlib/axes/_axes.py

```diff
@@ -3910,10 +3910,13 @@ def bxp(self, bxpstats, positions=None, widths=None, vert=True,
 
         zdelta = 0.1
 
-        def line_props_with_rcdefaults(subkey, explicit, zdelta=0):
+        def line_props_with_rcdefaults(subkey, explicit, zdelta=0,
+                                       no_marker=True):
```

Comment:
I'd either use `no_marker=False` as default or `marker=False`, so that the overriding calls would be `no_marker=True` or `marker=True` (depending on where you would like to override the default). The current overriding `no_marker=False` is unnecessarily difficult to follow because of the double negation.

Reply:
or `use_marker=False`

Reply:
Thanks for the feedback. Pushed an update.

Reply:
I think `use_no_marker` feels better . What do you think?
```python
if use_no_marker:
    d['marker'] = ''
```

Reply:
How is use_no_marker better than no_marker?

Reply:
I was going for a more verbose name for clarity but realized that it is unnecessary (going back to `no_marker`).

Reply:
The point is that `use_marker` is a positive action, whereas `no_marker` is a negative.  It is strange to have a default not not do something instead of just saying `use_marker=True`

Reply:
```python
if no_marker:
    d['marker'] = ''
```
(my reasoning:) since we are overriding a dictionary value, it feels better to not do it by default.
 I would leave it up to you and other core devs to take the final call. Thanks!

Reply:
If not use_marker:

Reply:
knew I could use a `not` but was waiting for you to take the final call on the name :). pushed a change. (Also, I see how this is a better choice. Thanks for the nudging me in the right direction).